### PR TITLE
[Expert] Recipes for CC:Tweaked and AdvancedPeripherals

### PR DESCRIPTION
**Pull request by mcilhargey**
Expertify most of the recipes from CC:Tweaked and Advanced Peripherals

Recipe examples:

computer
computer

advanced computer
advanced_computer

monitor
monitor

modem
modem

turtle
turtle

pocket computer
pocketcomputer

peripheral casing
peripheral_casing

chunk controller
chunk_controller

weak automata core
weak_automata_core

overpowered automata core
overpowered_weak_automata_core

### CODE REVIEW

File: kubejs/client_scripts/expert/item_modifiers/jei_hide_items.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ onEvent('jei.hide.items', (event) => {
         return;
     }
     let items = [
+        'advancedperipherals:me_bridge',
+        'advancedperipherals:colony_integrator',
```
Comment (NillerMedDild) [inline kubejs/client_scripts/expert/item_modifiers/jei_hide_items.js:7]: Please add these here instead: https://github.com/NillerMedDild/Enigmatica6/blob/master/kubejs/client_scripts/constants.js#L103
Those items are hidden regardless of mode

File: kubejs/client_scripts/normal/item modifiers/jei_hide_items.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        return;
+    }
+    let items = [
+        'advancedperipherals:me_bridge',
+        'advancedperipherals:colony_integrator'
+    ];
+    items.forEach((entry) => {
+        event.hide(entry);
+    });
+});
```
Comment (NillerMedDild) [inline kubejs/client_scripts/normal/item modifiers/jei_hide_items.js:12]: Please add these here instead: https://github.com/NillerMedDild/Enigmatica6/blob/master/kubejs/client_scripts/constants.js#L103
Those items are hidden regardless of mode

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    */
+
+    const recipes = [
+        {
+            output: 'advancedperipherals:peripheral_casing',
+            pattern: ['CDC', 'BAB', 'CEC'],
+            key: {
+                A: 'rftoolsbase:machine_frame',
+                B: 'create:redstone_link',
+                C: 'immersiveengineering:sheetmetal_iron',
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js:24]: I believe there's a tag version of this :)

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+            },
+            id: 'advancedperipherals:peripheral_casing'
+        },
+        {
+            output: 'advancedperipherals:chunk_controller',
+            pattern: ['BCB', 'DAE', 'BCB'],
+            key: {
+                A: 'advancedperipherals:peripheral_casing',
+                B: 'dustrial_decor:anchor',
+                C: 'minecraft:redstone_block',
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js:36]: Please use `#forge:storage_blocks/redstone`

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+                D: 'minecraft:respawn_anchor',
+                E: 'mekanism:upgrade_anchor'
+            },
+            id: 'advancedperipherals:chunk_controller'
+        },
+        {
+            output: 'advancedperipherals:weak_automata_core',
+            pattern: ['ABA', 'CDC', 'AEA'],
+            key: {
+                A: 'emendatusenigmatica:gold_gear',
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js:46]: Please use the tag `#forge:gears/gold` instead.

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+                B: 'advancedperipherals:peripheral_casing',
+                C: 'tconstruct:soul_glass',
+                D: 'quark:soul_bead',
+                E: 'modularrouters:player_module'
+            },
+            id: 'advancedperipherals:weak_automata_core'
+        },
+        {
+            output: 'advancedperipherals:end_automata_core',
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js:55]: Have you verified that this recipe works? :)

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/computercraft/shaped.js [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+                B: 'immersiveengineering:slab_sheetmetal_gold',
+                C: 'resourcefulbees:rgbee_honeycomb'
+            },
+            id: 'computercraft:computer_advanced'
+        },
+        {
+            output: 'computercraft:wired_modem',
+            pattern: [' A ', ' B ', ' C '],
+            key: {
+                A: 'immersiveengineering:sheetmetal_aluminum',
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/computercraft/shaped.js:45]: Please use a tag instead if possible

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/computercraft/shaped.js [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+                C: 'resourcefulbees:rgbee_honeycomb'
+            },
+            id: 'computercraft:monitor_advanced'
+        },
+        {
+            output: Item.of('computercraft:cable', 8),
+            pattern: [' B ', 'BAB', ' B '],
+            key: {
+                A: 'immersiveengineering:wirecoil_redstone',
+                B: 'pneumaticcraft:plastic'
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/computercraft/shaped.js:75]: Should this have been the plastic tag?

**Review by NillerMedDild — CHANGES_REQUESTED** [review]
I like these recipes a lot, great work :D

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js (hunk 6 above)
Comment (mcilhargey) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/advancedperipherals/shaped.js:55]: Yes, this recipe works fine

**Review by NillerMedDild — APPROVED** [review]

**Pull request merged**